lib/apr/views/commerce/commerce_order_slack_view.ex

```diff
@@ -24,6 +24,9 @@ defmodule Apr.Views.CommerceOrderSlackView do
       {"high_risk", "approved", %{"items_total_cents" => cents}}
         when cents >= 10_000_00 ->
           generate_slack_message(event, routing_key)
+      {"high_risk_async_payment", "processing_approval", %{"items_total_cents" => cents}}
```

Comment:
@starsirius I just wanted to let you know that we decided to have a new theme called `high_risk_async_payment` for the notification. Let me know if it's not what we want.

Reply:
@narikazu Apologies for the delay. Given the intent is the same with credit cards, I'd suggest we keep using the existing "high_risk" theme vs. creating a new one, to simplify things. Curious how you think?

Reply:
Totally makes sense! I've created a PR to address it: https://github.com/artsy/aprd/pull/380